Issue with step 6 of the 🚀 Quick Start Guide

In the Quick Start Guide in the README.me the 6th step poses an issue.

Currently, if you run 
```bash
cd docs
mdbook serve
```

You will come across this error
```bash
2025-04-01 13:39:07 [ERROR] (mdbook::utils): Error: Summary parsing failed for file="/Users/.../docs/src/SUMMARY.md"
2025-04-01 13:39:07 [ERROR] (mdbook::utils): 	Caused By: Duplicate file in SUMMARY.md: "./GettingStarted/QuickStart.md"
```

It is due to a duplicate Quick Start Guide guide in the `SUMMARY.md`

<img width="408" alt="Image" src="https://github.com/user-attachments/assets/6d34c7ac-65aa-4852-96df-2b6902788f87" />

It seems that the mdbook wants only one of them. I removed one of the guides from the markdown and then the `mdbook serve` command worked.  I'm not sure if it's due to developer error on my end, or if it's something that needs fixing.

--- a/docs/src/SUMMARY.md
+++ b/docs/src/SUMMARY.md
@@ -6,9 +6,8 @@
 ---
 
 - [Guides](./GettingStarted/README.md)
-    - [Quick Start](./GettingStarted/QuickStart.md)
-    - [Become a Provider](./GettingStarted/BecomeProvider.md)
-    - [Single Sign On](./GettingStarted/SSO.md)
+  - [Become a Provider](./GettingStarted/BecomeProvider.md)
+  - [Single Sign On](./GettingStarted/SSO.md)
 - [Fundamentals](./Fundamentals/README.md)
   - [Blockchain Basics](./Fundamentals/BlockchainBasics.md)
   - [Frequency Networks](./Fundamentals/Networks.md)

